### ISSUE TITLE
broken `Forwarding commands to existing instance` feature

### ISSUE THREAD

**Issue by illwieckz** [DaemonEngine/Daemon#78]
the “_Existing instance found / Forwarding commands to existing instance_” feature looks to be broken on `for-0.51.0` branch.

**Comment by illwieckz** [comment]
With command in the `+command` form it says it forwards command but nothing happens:

```
$ ./daemon +connect gg.illwieckz.net
Unvanquished 0.52.1 Linux x86_64 Jun 18 2021
cmdline: +connect gg.illwieckz.net
[FS] Lib path: /home/illwieckz/.local/share/unvanquished/base 
[FS] Home path: /home/illwieckz/.local/share/unvanquished 
[FS] Pak search path: /home/illwieckz/.local/share/unvanquished/base/pkg 
[FS] Pak search path: /home/illwieckz/.local/share/unvanquished/pkg 
Existing instance found 
Forwarding commands to existing instance 
```

But this is even worse with command in the `-command` form, the engine even wrongly assumes there is no command at all:

```
$ ./daemon -connect gg.illwieckz.net
Unvanquished 0.52.1 Linux x86_64 Jun 18 2021
cmdline: -connect gg.illwieckz.net
[FS] Lib path: /home/illwieckz/.local/share/unvanquished/base 
[FS] Home path: /home/illwieckz/.local/share/unvanquished 
[FS] Pak search path: /home/illwieckz/.local/share/unvanquished/base/pkg 
[FS] Pak search path: /home/illwieckz/.local/share/unvanquished/pkg 
Existing instance found 
No commands given, exiting... 
```

**Comment by bmorel** [comment]
Confirmed that the regression still happens as of 0.52.1

**Comment by illwieckz** [comment]
I don't know what fixed it but on my end it works with client running latest `0.53.0/sync` branch (even if forwarding client is `0.52.1`).

**Comment by illwieckz** [comment]
I only compared released `0.52.1` (no working) with self-build `master` and `0.53.0/sync` (not working), so it's also possible the way we build our release breaks the feature.

**Comment by illwieckz** [comment]
@slipher It looks like forwarding on Linux is broken when building release build (I haven't tested other systems).

I built `0.53.0/sync` with our release docker recipe and forwarding is broken, while forwarding works when I build the engine myself.
